01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,16 @@
+def fizzbuzz(sum)
+sum += 1
```

Comment:
メソッドの中では一段インデントしましょう。

```suggestion
  sum += 1
```